app/templates/_service_meta.html

```diff
@@ -29,12 +28,10 @@ <h2 class="visuallyhidden">Service documents</h2>
   </ul>
   <h2 class="sidebar-heading">Framework</h2>
   {% if service.meta.externalFrameworkUrl is not none %}
-    {%
-      with
-      text=service.frameworkName,
-      link=service.meta.externalFrameworkUrl %}
-      {% include "toolkit/external-link.html" %}
-    {% endwith %}
+    <a href="{{ service.meta.externalFrameworkUrl }}"
```

Comment:
This is an interesting one - we haven't linked to the CCS framework agreement URL since G-Cloud 7. While we still show expired service pages, the [G-Cloud 7 agreement link](https://www.crowncommercial.gov.uk/agreements/rm1557vii) now 404s 😿 

So I think we can safely simplify this section and just have

```
<h2 class="sidebar-heading">Framework</h2>
<p class="framework-name">{{ service.frameworkName }}</p>
```

Reply:
Awesome!